src/os/bluestore/BlueFS.h

```diff
@@ -236,6 +236,13 @@ class BlueFS {
     void skip(size_t n) {
       pos += n;
     }
+
+    void invalidate_cache(uint64_t offset, uint64_t length) {
+      if (offset >= bl_off && offset < get_buf_end()) {
+	bl.clear();
```

Comment:
May be worth leaving a comment saying that buffer is invalidated completely rather than for the provided extent. For the sake of simplicity I presume?

Reply:
@ifed01 . In fact, code existed so long and there is no bug occur. I think rocksdb do some protect. But from the theory, we should disable cache.

Reply:
@majianpeng - so you haven't seen any real issue caused by the lack of this invalidation, have you?

Reply:
Yes.